ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -377,9 +373,61 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
         loading: false,
         showUnderservedPopulationDetails: false,
         showStigmatizationDetails: false,
+        billingAccounts: []
       };
     }
 
+    async componentDidMount() {
+      if (serverConfigStore.getValue().enableBillingLockout) {
```

Comment:
nit: pretty heavy nesting. I might at least break out this first case into an inverted flag check + early return to eliminate one level.

Reply:
I pulled it out into a function and I think that helped a little. I tried shuffling around the stuff inside but couldn't find a format that I like more than what I already have.